Core/clim-basic/ports.lisp

```diff
@@ -76,8 +76,13 @@
                         :reader port-text-style-mappings)
    (pointer-sheet :initform nil :accessor port-pointer-sheet
 		  :documentation "The sheet the pointer is over, if any")
+   ;; The difference between grabbed-sheet and pressed-sheet is that
```

Comment:
Commit message:
- "(so port may simply specialize on port)" Not sure what that means
* "Corrected coordinate fixup (get-pointer-position was buggy)" Is this right? I don't see changes to `get-pointer-position`
* "- Pressed sheet implementation (i.e for scrolling)\- Grabbed sheet implementation" Those are not new, but restored (they were removed in 23b66e37).
* "synthetized" -> "synthesized" according to my spell checker
* "Pressed sheet is activated when …" -> "The `pressed-sheet` slot is bound to a sheet when …"
* "with a macro WITH-POINTER-GRABBED" -> "with the macro WITH-POINTER-GRABBED"

Reply:
"(so port may simply specialize on port)" Not sure what that means

So `a port` may subclass `clim:port` and specialize the protocol methods.

Reply:
fixed